### PR TITLE
Add integration test check with enabled analyzer

### PR DESCRIPTION
**Pull request by novikd**
### Changelog category (leave one):

- Build/Testing/Packaging Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Add integration test check with the enabled analyzer.

### CODE REVIEW

File: tests/integration/helpers/cluster.py [hunk 1]
```diff
@@ -4187,6 +4187,8 @@ def write_embedded_config(name, dest_dir, fix_log_level=False):
             )
 
         write_embedded_config("0_common_instance_users.xml", users_d_dir)
+        if os.environ.get('USE_NEW_ANALYZER') is not None:
```
Comment (novikd) [inline tests/integration/helpers/cluster.py:4190]:
```suggestion
        if os.environ.get("USE_NEW_ANALYZER") is not None:
```

File: tests/ci/integration_test_check.py [hunk 2]
```diff
@@ -226,6 +226,8 @@ def main():
         download_all_deb_packages(check_name, reports_path, build_path)
 
     my_env = get_env_for_runner(build_path, repo_path, result_path, work_path)
+    if "analyzer" in check_name.lower():
```
Comment (Felixoid) [inline tests/ci/integration_test_check.py:229]: It looks like a part of `get_env_for_runner` to me

**Review by Felixoid — APPROVED** [review]
Looks good. Do you plan to leave it *only* for pull_request.yml? There are different configs in .github/workflows

**Comment by novikd** [comment]
@Felixoid I want to add it to master too, but for development purposes, I started with only pull requests. It's suspicious that all tests passed, I need to check in logs if the analyzer was actually enabled.

**Pull request merged**